packages/cli/src/commands/lint.js

```diff
@@ -21,9 +23,19 @@ export const builder = (yargs) => {
 }
 
 export const handler = ({ fix }) => {
-  execa('yarn eslint', [fix && '--fix', 'web/src', 'api/src'].filter(Boolean), {
-    cwd: getPaths().base,
-    shell: true,
-    stdio: 'inherit',
-  })
+  const paths = getPaths()
+  const webExists = fs.existsSync(paths.web.src)
+  const apiExists = fs.existsSync(paths.api.src)
+
+  execa(
+    'yarn eslint',
+    [fix && '--fix', webExists && 'web/src', apiExists && 'api/src'].filter(
+      Boolean
+    ),
+    {
+      cwd: paths.base,
+      shell: true,
+      stdio: 'inherit',
+    }
+  )
```

Comment:
```suggestion
  execa(
    'yarn eslint',
    [
      fix && '--fix',
      fs.existsSync(getPaths().web.src) && 'web/src',
      fs.existsSync(getPaths().api.src) && 'api/src',
    ].filter(Boolean),
    {
      cwd: getPaths().base,
      shell: true,
      stdio: 'inherit',
    }
  )
```

This is silly, but because doing it this way makes prettier keep the array elements on separate lines, I like this more.

Reply:
No problemo